cmd/webhook/main.go

```diff
@@ -91,7 +91,8 @@ var ourTypes = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
 	sourcesv1alpha2.SchemeGroupVersion.WithKind("SinkBinding"):     &sourcesv1alpha2.SinkBinding{},
 	sourcesv1alpha2.SchemeGroupVersion.WithKind("ContainerSource"): &sourcesv1alpha2.ContainerSource{},
 	// v1beta1
-	sourcesv1beta1.SchemeGroupVersion.WithKind("SinkBinding"): &sourcesv1beta1.SinkBinding{},
+	sourcesv1beta1.SchemeGroupVersion.WithKind("SinkBinding"):      &sourcesv1beta1.SinkBinding{},
+	sourcesv1beta1.SchemeGroupVersion.WithKind("ContainerSource"):  &sourcesv1beta1.SinkBinding{},
```

Comment:
should be &sources.v1beta1.ContainerSource{}

Reply:
Oops, corrected